client-failover/client-failover-sample-hazelcast5/src/main/java/sample/com/hazelcast/clientfailover/model/City.java

```diff
@@ -0,0 +1,14 @@
+package sample.com.hazelcast.clientfailover.model;
+
+import com.hazelcast.core.HazelcastJsonValue;
+import com.hazelcast.org.json.JSONObject;
+
+public final class City {
+    public static HazelcastJsonValue asJson(String country, String city, int population) {
```

Comment:
I see that we have shortened the example, and no longer use this file. Can you please remove it, and any references to it?